src/internal/data-service-api/data-service-dialects-api.ts

```diff
@@ -30,7 +30,7 @@ export class DataServiceDialectsApiClient implements DataServiceDialectsApi {
     const token = await this.tokenProvider.get();
     return withReThrowingDataServiceError(
       axios
-        .post<DialectAccountDto>(`${this.baseUrl}/api/v1/dialects`, command, {
+        .post<DialectAccountDto>(`${this.baseUrl}/v0/dialects`, command, {
```

Comment:
Changing these URLs was the only way I could get the tests to work - let me know if you want to change these back @tsmbl

Reply:
@kevindingens, seems like you have old data service version, should be v1 here
Could you please verify that you run last data-service version from main branch?

Reply:
Yes I pulled late last night to test and I pulled a few minutes ago. Same thing. Here is a screenshot of the Swagger
<img width="714" alt="Screen Shot 2022-06-23 at 10 17 02 AM" src="https://user-images.githubusercontent.com/64697796/175334279-81b2fd12-6332-41e1-b2ed-1a7b2302c454.png">